includes/class-give-donate-form.php

```diff
@@ -567,10 +567,11 @@ public function get_minimum_price() {
 	public function get_maximum_price() {
 
 		if ( ! isset( $this->maximum_price ) ) {
-			$this->maximum_price = give_get_meta( $this->ID, '_give_custom_amount_range_maximum', true );
+			$maximum_price       = 999999.99;
+			$this->maximum_price = give_get_meta( $this->ID, '_give_custom_amount_range_maximum', true, $maximum_price );
 
 			if ( ! $this->is_custom_price_mode() ) {
-				$this->maximum_price = 999999.99;
+				$this->maximum_price = $maximum_price;
```

Comment:
@raftaar1191 try to use `give_get_highest_price_option` when custom amount disabled and let me know.

Reply:
Sure @ravinderk updating the PR